### DISCUSSION TITLE
Notebook-style navigation and better text editing support (like onenote's)

### DISCUSSION THREAD

**Discussion by olbotta** [xournalpp/xournalpp#3690] [Ideas] [answered]
i would really like to switch from Onenote to xournalpp but there are 2 key features i am missing:

- **notebook style navigation**: the ability to navigate between different pages in the same UI. Separating different topics in "folders" and sub-folders eases the studying and notetaking process.

- **better text editing**: i take handwritten notes but also a lot of keyboard-written. it would be nice to have a support for multiple styles (different headings, titles), bulleted lists, tables and so on. 
    maybe integrate markdown like joplin?

I am in a tremendous vendor lock-in from Microsoft since onenote is the only thing that is keeping me from switching definitely to GNU/linux.

thank you and have a nice day!

**Comment by rolandlo** [comment, marked as answer]
- for a discussion about notebook style navigation see issue #799
- for Markdown support there is a pull request #3408
